development/docker-compose.dev.yml

```diff
@@ -18,3 +18,14 @@ services:
     volumes:
       - "./nautobot_config.py:/opt/nautobot/nautobot_config.py"
       - "../:/source"
+  docs:
+    entrypoint: "mkdocs serve -v -a 0.0.0.0:8080"
+    ports:
+      - "8001:8080"
+    volumes:
+      - "../docs:/source/docs:ro"
+      - "../mkdocs.yml:/source/mkdocs.yml:ro"
```

Comment:
```suggestion
      - "../:/source"
```

Reply:
Change from the [cookiecutter PR](https://github.com/networktocode-llc/cookiecutter-ntc/pull/245/files#diff-111e741490805ab2512acd43ce3d860fd58fd9164b7ae79ff58ba6aa6b6ba3cb). It avoids various issues and the previous bind mounts were unnecessary.